composer.json

```diff
@@ -29,6 +29,12 @@
         "webflo/drupal-finder": "^1.0.0",
         "webmozart/path-util": "^2.3",
         "drupal/core-recommended": "^8.8",
+        "drupal/fast_404":"1.0.0-alpha4",
```

Comment:
These changes are against the PR #131  https://github.com/govCMS/govcms8lagoon/pull/131/files